src/modules/chat/chat.service.ts

```diff
@@ -39,9 +55,31 @@ export class ChatService {
     }
 
     const userChats = user.chatRooms;
+    const EQUAL_SORT_ORDER = 0;
+
+    const sortedUserChats = await Promise.all(
```

Comment:
why you dont sort it in the query?

Reply:
but you need you use query builder, I'm not sure it possible with repository

Reply:
Agree. Fixed